python/ray/train/_internal/logging.yaml

```diff
@@ -0,0 +1,36 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+  ray:
+    format: "%(asctime)s\t%(levelname)s %(filename)s:%(lineno)s -- %(message)s"
```

Comment:
Note: this is the default ray logging format. See here: https://docs.ray.io/en/latest/ray-observability/user-guides/configure-logging.html#system-logs